vitest.config.ts

```diff
@@ -0,0 +1,8 @@
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  test: {
+    // Don't match "*.spec.ts" (like the vitest default does). They are for playwright.
+    include: ['**/*.test.?(c|m)[jt]s?(x)'],
```

Comment:
Can we put tests file in `./tests`? Also, what's `?` after `.` for?

Reply:
`./tests` on the root level or in `packages/waku`?

And that brings another question:

I added `vitest` and `ts-expect` to the root level, since there is the `test` command, but I'm not sure if it would make more sense to have them in `packages/waku` only. Also not sure where the test fail comes from, but it smells like its related to directories.

Reply:
> ./tests on the root level or in packages/waku?

I prefer `packages/waku/tests`.

> I added vitest and ts-expect to the root level, since there is the test command, but I'm not sure if it would make more sense to have them in packages/waku only.

Not sure either. @Aslemammad @himself65 thoughts?

Reply:
packages/waku/tests seems good to me